nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/PayaraHazelcastTenantFactory.java

```diff
@@ -66,7 +69,7 @@ public TenantControl saveCurrentTenant() {
         if (invocation != null) {
             tenantControl = invocation.getRegistryFor(TenantControl.class);
             if (tenantControl == null && ctxUtil.isInvocationLoaded()) {
-                blockingDisabled = Boolean.getBoolean(DISABLE_BLOCKING_PROPERTY);
+                blockingDisabled = getDisableBlockingProperty.get();
```

Comment:
Why is the static variable overwritten here?

Reply:
So it's possible to change the system property without restarting the instance and having it take effect